synapse/exc.py

```diff
@@ -125,6 +125,14 @@ class FileExists(SynErr):pass
 class NotEmpty(SynErr):pass
 class NotSupported(SynErr):pass
 
+class DatabaseInconsistent(SynErr):
+    ''' If you get this Exception, that means the database is corrupt '''
+    pass
+
+class DatabaseLimitReached(SynErr):
```

Comment:
maybe we should rename this HitCoreLimit() and include name=limitname  size=limitsize as well as mesg ( and move it up by the other HitFooLimit() stuff )